### PR TITLE
Add a footer to the app

### PR DESCRIPTION
**Pull request by bp323**
We need to add a footer to the app that displays the following:

Admin UI:
- Privacy and cookie policy (http://www.admin.cam.ac.uk/cookie.html)
- FAQ (TBA)
- Help desk details (http://www.ucs.cam.ac.uk/support/service-desk/)
- Help email (service-desk@ucs.cam.ac.uk)

Student UI:
- Privacy and cookie policy (http://www.admin.cam.ac.uk/cookie.html)
- FAQs and useful documentation (TBA)
- Help email (service-desk@ucs.cam.ac.uk)

As a design we got a poorly constructed footer used in project light:
image001

### CODE REVIEW

**Comment by bp323** [comment]
Added the full footer everywhere instead of a slimmed down version per email. If something isn't supposed to be added they can let us know or create a ticket for it.

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: apps/admin/ui/index.html [hunk 1]
```diff
@@ -51,6 +51,35 @@
             <div class="clearfix"></div>
         </div>
 
+        <footer id="footer-help">
+            <div id="footer-help-logo" class="pull-left">
+                <a href="javascript:;" id="gh-header-logo" target="_blank" title="Visit Timetable">
+                    <img src="/shared/gh/img/unilogo.png" alt="Cambridge University">
+                </a>
+            </div>
+
```
Comment (Coenego) [inline apps/admin/ui/index.html]: This empty line can probably go

File: apps/timetable/admin/app/index.html [hunk 2]
```diff
@@ -65,6 +65,35 @@
             <div class="clearfix"></div>
         </div>
 
+        <footer id="footer-help">
+            <div id="footer-help-logo" class="pull-left">
+                <a href="javascript:;" id="gh-header-logo" target="_blank" title="Visit Timetable">
+                    <img src="/shared/gh/img/unilogo.png" alt="Cambridge University">
+                </a>
+            </div>
+
```
Comment (Coenego) [inline apps/timetable/admin/app/index.html]: Same for this one

**Comment by Coenego** [comment]
Apart from some minor issues and the issue with the collapsed modules, it seems ok. Back to @bp323 for follow-up.

**Comment by bp323** [comment]
Followed up on review, back to @Coenego

**Pull request merged**